library/public/camera.h

```diff
@@ -73,6 +73,9 @@ class F3D_EXPORT camera
 
   ///@}
 
+  virtual camera& rotateByMouse(double deltaX, double deltaY) = 0;
+  virtual camera& rotateAroundFocalPoint(double deltaX, double deltaY) = 0;
```

Comment:
I'm not sure to follow how that relates to #1419

Reply:
That's also what I realized. This was just a feature that seemed cool to me. I will implement something later.

Reply:
That's ok,  you can add new features if you want, but please discuss them in discord/ a dedicated issue first :)

Reply:
Yeah my bad sorry. I was working on this separetely but PRs got mixed. I will separate them tonight.